Sdk/Speckle.Connectors.Common/Connector.cs

```diff
@@ -24,8 +24,8 @@ HostAppVersion version
     TypeLoader.Initialize(typeof(Base).Assembly, typeof(Point).Assembly);
 
     var (logging, tracing) = Observability.Initialize(
+      application.Name + " " + HostApplications.GetVersion(version),
```

Comment:
changing the order of properties was intentional or not?

Reply:
Intentional in that I just put would I thought was most significant to least.  But I did go through some refactorings